Neos.Flow/Classes/Http/Helper/RequestInformationHelper.php

```diff
@@ -68,6 +68,7 @@ public static function getScriptRequestPath(ServerRequestInterface $request): st
      */
     public static function getRelativeRequestPath(ServerRequestInterface $request): string
     {
+        // TODO: How to fix this? Probably give base URI to method, but how to get it in usages.
```

Comment:
Why do we need this method, where is it used?

Reply:
In Routing ;)
\Neos\Flow\Mvc\Routing\Route::matches
\Neos\Flow\Mvc\Routing\Dto\RouteContext::getCacheEntryIdentifier
\Neos\Flow\Mvc\Routing\RouterCachingService::storeMatchResults

Reply:
After our discussion and re-reading the code it makes only sense to build a "base URI" from the given request, everything else is wrong anyway, so that's changed.